src/index.js

```diff
@@ -115,6 +116,20 @@ module.exports = async function run() {
       }
     }
 
+    // Ignore errors if specified labels are added.
+    if (ignoreLabels && validationError) {
+      const labelNames = pullRequest.labels.map((label) => label.name);
+      for (const labelName of labelNames) {
+        if (ignoreLabels.includes(labelName)) {
+          core.info(
+            `Validation was skipped because the PR label "${labelName}" was found.`
+          );
+          validationError = null;
+          break;
+        }
+      }
+    }
```

Comment:
This code block looks good! In my other comment I meant if we can move this before trying to detect a `validationError` in the first place. We don't need to execute the loop where we fetch all commits and run the validation if we already know that we're going to ignore it.

Does that make sense?

Reply:
Yes, it makes sense. But I'm not sure how to do an early exit with GitHub Actions. I'll give it a try.

Reply:
Oh, it worked with just `return`. 2ad53dea3eb6399271a17071cb6d741f02e126c2
https://github.com/kenji-miyake/action-semantic-pull-request/runs/5992394229?check_suite_focus=true#step:6:8